cve_bin_tool/cli.py

```diff
@@ -6,8 +6,7 @@
 This tool scans for a number of common, vulnerable open source components
 (openssl, libpng, libxml2, expat and a few others) to let you know if your
 system includes common libraries with known vulnerabilities.  It emits a list
-of CVE numbers that may be relevant to your binary based on the versions.  It
-cannot detect backported fixes.
+of CVE numbers that may be relevant to your binary based on the versions.
```

Comment:
I wouldn't remove this entirely -- it's still relevant when we don't know which distro you got your stuff from, or if 
 you got it from something like a Yocto layer, or if anything was manually backported by a team, or for basically anything on windows.  Honestly, I suspect the cases where we don't detect backported fixes exceed the ones where we do.

Maybe something like "There is a flag to enable information about backported fixes for specific Linux distributions, but cve-bin-tool cannot detect backported fixes outside of those published lists."

Reply:
Yeah, that makes more sense than deleting the sentence.